helix-term/src/ui/document.rs

```diff
@@ -323,6 +324,7 @@ pub struct TextRenderer<'a> {
     pub space: String,
     pub tab: String,
     pub tab_width: u16,
+    pub indent_width: IndentStyle,
```

Comment:
Since indent_width is cast to a `usize` in all locations inside `draw_indent_guides`, wouldn't it make more sense to store indent_width as a `usize` instead of `u16`?

Reply:
It probably doesn't matter too mich but I usually try to store values in the smallest possible integer types. No neccisairly for performamce (doesn't really matter either way likely) but because upwards casting can neber overflow while downards casting can so I would rather store a smaller type and upwards cast than the reverse. It's not really documented but tab width is sort of unofficially limited to u16 so that's why I would.use that